pkg/cmd/release-payload-controller/payload_accepted_controller.go

```diff
@@ -0,0 +1,160 @@
+
+	releasePayloadInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: c.Enqueue,
+		UpdateFunc: func(oldObj, newObj interface{}) {
+			c.Enqueue(newObj)
+		},
+		DeleteFunc: c.Enqueue,
+	})
+
+	return c, nil
+}
+
+func (c *PayloadAcceptedController) Enqueue(obj interface{}) {
+	key, err := cache.MetaNamespaceKeyFunc(obj)
+	if err != nil {
+		utilruntime.HandleError(fmt.Errorf("invalid queue key '%v': %v", obj, err))
+		return
+	}
+	c.queue.Add(key)
+}
+
+func (c *PayloadAcceptedController) Run(ctx context.Context) {
+	defer utilruntime.HandleCrash()
+
+	klog.Info("Starting Payload Accepted Controller")
+	defer func() {
+		klog.Info("Shutting down Payload Accepted Controller")
+		c.queue.ShutDown()
+		klog.Info("Payload Accepted Controller shut down")
+	}()
+
+	if !cache.WaitForNamedCacheSync("PayloadAcceptedController", ctx.Done(), c.cachesToSync...) {
+		return
+	}
+
+	go func() {
+		wait.UntilWithContext(ctx, c.runWorker, time.Second)
+	}()
+
+	<-ctx.Done()
+}
+
+func (c *PayloadAcceptedController) runWorker(ctx context.Context) {
+	for c.processNextItem(ctx) {
+	}
+}
+
+func (c *PayloadAcceptedController) processNextItem(ctx context.Context) bool {
+	key, quit := c.queue.Get()
+	if quit {
+		return false
+	}
+	defer c.queue.Done(key)
+
+	err := c.sync(ctx, key.(string))
+
+	if err == nil {
+		c.queue.Forget(key)
+		return true
+	}
+
+	utilruntime.HandleError(fmt.Errorf("%v failed with : %w", key, err))
+	c.queue.AddRateLimited(key)
+
+	return true
+}
+
+func (c *PayloadAcceptedController) sync(ctx context.Context, key string) error {
+	klog.V(4).Infof("Starting PayloadAcceptedController sync")
+	defer klog.V(4).Infof("PayloadAcceptedController sync done")
+
+	// Always start 10 seconds later after a change occurred. Makes us less likely to steal work and logs from the operator.
+	timer := time.NewTimer(10 * time.Second)
+	defer timer.Stop()
+	select {
+	case <-timer.C:
+	case <-ctx.Done():
+		return nil
+	}
+
+	// Convert the namespace/name string into a distinct namespace and name
+	namespace, name, err := cache.SplitMetaNamespaceKey(key)
+	if err != nil {
+		utilruntime.HandleError(fmt.Errorf("invalid resource key: %s", key))
+		return nil
+	}
+
+	// Get the ReleasePayload resource with this namespace/name
+	releasePayload, err := c.releasePayloadLister.ReleasePayloads(namespace).Get(name)
+	if err != nil {
+		// The ReleasePayload resource may no longer exist, in which case we stop processing.
+		if errors.IsNotFound(err) {
+			utilruntime.HandleError(fmt.Errorf("releasePayload '%s' in work queue no longer exists", key))
+			return nil
+		}
+
+		return err
+	}
+	klog.Infof("Syncing Payload Accepted for ReleasePayload: %s/%s", releasePayload.Namespace, releasePayload.Name)
+	updated := releasePayload.DeepCopy()
+
+	if _, err := c.releasePayloadClient.ReleasePayloads(namespace).Update(ctx, updated, v1.UpdateOptions{}); err != nil {
```

Comment:
do a check of deepequal(updated.status, original.status).  don't update if nothing changed.